index.html

```diff
@@ -1,155 +1,165 @@
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Dienebi Ombu - Portfolio</title>
-    <link rel="stylesheet" href="styles.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
-  </head>
-  <body>
-    <header>
-      <nav class="toolbar">
-        <a class="logo" href="index.html">ebidien</a>
-        <ul class="menu-items">
-          <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
-          <li><a href="#about" class="menu-links">About</a></li>
-          <li><a href="#contact" class="menu-links">Contact</a></li>
-        </ul>
-        <img class="main-menu" src="menu-button.svg" alt="Main Menu" />
-      </nav>
-    </header>
-    <section class="headline">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Dienebi Ombu - Portfolio</title>
+  <link rel="stylesheet" href="styles.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+
+<body>
+  <header>
+    <nav class="toolbar">
+      <a class="logo" href="index.html">ebidien</a>
+      <ul class="menu-items">
+        <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
+        <li><a href="#about" class="menu-links">About</a></li>
+        <li><a href="#contact" class="menu-links">Contact</a></li>
+      </ul>
+      <img class="main-menu" src="menu-button.svg" alt="Main Menu" />
+    </nav>
+  </header>
+  <section class="headline">
+    <div>
+      <div class="intro-title">
+        <h1>I'm Dienebi. Glad to see you!</h1>
+      </div>
+      <div class="intro-body">
```

Comment:
- [ ] If you notice from the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=K43WlZW7LKElMPUk-0), there is a [border-bottom-left-radius](https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom-left-radius#:~:text=The%20border%2Dbottom%2Dleft%2D,the%20curvature%20of%20the%20corner.). Please ensure to apply this to ensure that the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#project-requirements) are met. Consider the attached screenshot for reference ☺️ 

- Your design 😎 
![curv](https://user-images.githubusercontent.com/99394374/213885543-90b477f4-e8d3-4c60-9816-f60100bfd0b1.PNG)